### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by snowme34**

### CODE REVIEW

**Comment by snowme34** [comment]
It seems that converting this repo into organization breaks the travis CI?

**Comment by snowme34** [comment]
I will squash this commit since the commit history is not that meaningful.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  #   - master
+  except:
+  - legacy
+  - experimental
+sudo: false
+before_install:
+  - nvm install node
+  - npm --version
+  # need to manually update the version (if needed)
+  # https://yarnpkg.com/lang/en/docs/install-ci/
```
Comment (snowme34) [inline .travis.yml]: need to manually update the version

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+dist: trusty
+language: minimal
+branches:
+  # only:      # will be useful if we have limited plan someday
```
Comment (snowme34) [inline .travis.yml]: If we don't want to run the checks for some specific branches in the future

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.15.2
+  - export PATH=$HOME/.yarn/bin:$PATH
+cache:
+  yarn: true
+  directories:
+    - node_modules
+install:
+  # - rm -rf ./node_modules # clean up if don't want cache
+  - yarn --version
+script:
```
Comment (snowme34) [inline .travis.yml]: If we add tests for other components, might need to update the commands here

**Comment by snowme34** [comment]
It should be ready to merge. The basic unit test for front-end will work as expected.

But beware that the config file might need to be updated soon if we want more tests.

**Comment by CTrando** [comment]
LGTM! Squash them when you merge.

**Pull request merged**